align/config/basic_template.sp

```diff
@@ -14,6 +14,14 @@ M0 (D G S S) NMOS w=w1 l=l1
 M0 (D G S S) PMOS w=w1 l=l1
 .ends Switch_PMOS
 
+.subckt Switch_NMOS_G  D G S
+M0 (D G S G) NMOS w=w1 l=l1
```

Comment:
@kkunal1408 , what is the intention for this primitive where gate and bulk terminals are shorted?

Reply:
We have separate generators for nmos with source and body shorted. So we had to create two mappings for a nmos. 
This one is another with gate and body shorted, Though we don't have a generator for this, so this is a workaround to map it back to Switch_NMOS after identifying.